Sources/SwiftCliCore/ChartConstructor.swift

```diff
@@ -8,112 +8,102 @@ public class ChartConstructor {
         let number_of_rows = stitch_array.count
         var finished_chart = ""
         let lastrow = number_of_rows - 1
-        // NEW LOGIC
+        
         for row in 0...lastrow {
             let row_length = stitch_array[row].count
             
             if (row == 0){
-                finished_chart =  self.make_bottom_row(width: row_length) + finished_chart
-                finished_chart = self.make_stitch_row(row: stitch_array[row]) + finished_chart
+                finished_chart =  makeBottomRow(width: row_length) + finished_chart
+                finished_chart = makeStitchRow(row: stitch_array[row]) + finished_chart
             }
             
             
             else {
                 let prev_row_length = stitch_array[row - 1].count
                 let right_side_diff = (row_length - prev_row_length)
                 if (right_side_diff < 0){
-                    print("here")
-                    finished_chart =  make_middle_row_stitch_count_change(width: row_length, right_difference: right_side_diff, left_difference: 0) + finished_chart
+                    
+                    finished_chart =  makeMiddleRowStitchCountChange(width: row_length, right_difference: right_side_diff, left_difference: 0) + finished_chart
                 }
                 else {
-                    finished_chart = self.make_middle_row(width: row_length) + finished_chart
+                    finished_chart = makeMiddleRow(width: row_length) + finished_chart
                 }
                 
-                finished_chart = self.make_stitch_row(row: stitch_array[row]) + finished_chart
-
-                }
+                finished_chart = makeStitchRow(row: stitch_array[row]) + finished_chart
+                
+            }
+            
             
-            //if last row
             if (row == (number_of_rows - 1)){
-                finished_chart = self.make_top_row(width: row_length) + finished_chart
-
+                finished_chart = makeTopRow(width: row_length) + finished_chart
+                
             }
-
+            
         }
         
         
         
         return(finished_chart)
     }
-    public func make_top_row(width: Int) -> String {
+    public func makeTopRow(width: Int) -> String {
         var top_row_bars = ""
         if (width > 0){
             top_row_bars += "┌"
             if (width > 1){
-                for _ in 1...(width - 1) {
-                    top_row_bars += "─┬"
-                }
+                top_row_bars += String(repeating: "─┬", count: width - 1)
+                
             }
             top_row_bars += "─┐"
         }
         top_row_bars += "\n"
```

Comment:
So if you want to try using string interpolation as I suggested in my previous review, you can make this code a lot easier to follow with something like:

```{swift}
switch width {
case 0:
    return "\n"
case 1:
    return "┌─┐\n"
default:
    middleBoxes = String(repeating: "─┬", count: width - 1)
    return "┌\(middleBoxes)─┐\n"
} 
```